disnake/member.py

```diff
@@ -650,6 +654,19 @@ def voice(self) -> Optional[VoiceState]:
         """Optional[:class:`VoiceState`]: Returns the member's current voice state."""
         return self.guild._voice_state_for(self._user.id)
 
+    @property
+    def communication_disabled_until(self) -> Optional[datetime.datetime]:
+        """Optional[:class:`datetime.datetime`]: Returns the datetime when the time out will be removed, if any
+
+        .. versionadded:: 2.3
+        """
+
+        if self._communication_disabled_until is not None:
+            if self._communication_disabled_until < utils.utcnow():
```

Comment:
```suggestion
        if self._communication_disabled_until and self._communication_disabled_until < utils.utcnow():
```

Reply:
Yeah, that way is better, implemented in 9450c709bcd723d04fe12e084431009d738c5cf7.